docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -0,0 +1,162 @@
+# Deployment guide: Guestbook sample application
+In this example, you will learn how you can use Terraform to create a Layer0 service as well as supporting infrastructure in the form of a persistent data store. This example does not cover the details of application being deployed but focuses on how you can combine Layer0 and other Terraform providers as part of a Terraform workflow.
+
+## Before you start
+In order to complete the procedures in this section, you must have following installed and configured correctly:
+
+ * Layer0
+ * Terraform
+ * Layer0 Terraform Provider
+
+Layer0 v0.8.4 or later installed and configured. If you have not already configured Layer0, see the [Layer0 installation guide](/setup/install). If you are running an older version of Layer0, see the [Layer0 upgrade instructions](/setup/upgrade#upgrading-older-versions-of-layer0).
+
+See the [Terraform installation guide](/reference/terraform-plugin#install) to install Terraform and the Layer0 Terraform Plugin.
+
+---
+
+## Deploy with Terraform
+Using Terraform, you will deploy a simple guestbook application, which is backed by AWS DynamoDb Table, for persistant storage. The terraform configuration file will use both the Layer0 and AWS Terraform providers, to deploy the guestbook application and provision a new DynamoDb Table.
+
+## Part 1: Clone the Layer0 examples repository
+Run this command to clone the `quintilesims/layer0-examples` repository
+
+<ul>
+  <li class="command">git clone https://github.com/quintilesims/layer0-examples.git</li>
+</ul>
+
+Inside a terminal window, navigate to the `terraform-beyond-layer0 folder`. You should find the following files once you are in the folder. We use these files to set up a Layer0 envrionment and deploy AWS resources with Terraform:
+
+|Filename|Purpose|
+|----|----|
+|[terraform.tfvars](https://github.com/quintilesims/layer0-examples/blob/master/guestbook-db/terraform.tfvars)|Variables specific to the environment and guestbook application|
+|[Dockerrun.aws.json](https://github.com/quintilesims/layer0-examples/blob/master/guestbook-db/Dockerrun.aws.json)|Template for running the guestbook application in a Layer0 environment|
+|[layer0.tf](https://github.com/quintilesims/layer0-examples/blob/master/guestbook-db/layer0.tf)|Provision Layer0 resources|
```

Comment:
This file also provisions the DynamoDB stuff, right? Maybe we should say something like "Provision Layer0 and AWS resources"?